tests/integration_tests/tests/agentless_tests/test_inter_deployment_dependencies.py

```diff
@@ -232,15 +238,17 @@ def _assert_dependencies_exist(self,
     def _get_dep_update_test_dependencies(self,
                                           is_first_state,
                                           should_keep_old_dependencies=False):
-        static_changed_to_static = SR_DEPLOYMENT1 if is_first_state \
-            else SR_DEPLOYMENT2
-        static_changed_to_runtime = SR_DEPLOYMENT1
-        runtime_changed_to_runtime = None
-        runtime_changed_to_static = None if is_first_state else SR_DEPLOYMENT2
-        shared_resource_target_id = SR_DEPLOYMENT1 if is_first_state \
-            else SR_DEPLOYMENT2
-        comp_target_id = COMP_DEPLOYMENT1 if is_first_state \
-            else COMP_DEPLOYMENT2
+        shared_deployment_target = SR_DEPLOYMENT1
+        comp_target_id = COMP_DEPLOYMENT1
+        if not is_first_state:
+            shared_deployment_target = SR_DEPLOYMENT2
+            comp_target_id = COMP_DEPLOYMENT2
+
+        static_changed_to_static = shared_deployment_target
+        static_changed_to_runtime = shared_deployment_target
+        runtime_changed_to_runtime = shared_deployment_target
+        runtime_changed_to_static = shared_deployment_target
+        shared_resource_target_id = shared_deployment_target
```

Comment:
i don't think we need 5 names for the same variable.. :D

Reply:
Done and updated